docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-space-manager.md

```diff
@@ -30,13 +32,13 @@ export PATH=$(pwd)/bin:$PATH
 ```
 Next deploy the space framework CRDs in the space management cluster.
 ```shell
-KUBECONFIG=$SM_CONFIG kubectl --context sm-mgt apply -f config/crds/
+KUBECONFIG=$SM_CONFIG kubectl apply -f config/crds/
 cd ..
 ```
 Finally, start the space-manager controller.
 
 ```shell
-space-manager --kubeconfig $SM_CONFIG --context sm-mgt -v 2 &
+space-manager --kubeconfig $SM_CONFIG --context sm-mgt -v 4 &> /tmp/space-manager.log &
```

Comment:
With the assumption in line 11 about the current context, do we still need to pass --context to space-manager?  I think the space-manager has a default of context of sm-mgt.

Reply:
I first tried omitting the `--context` and the space manager crashed with a complaint about not being able to use the context `kind-mgt`, if I recall correctly. Please look into the default behavior of the space manager.

Reply:
yes the SM doesn't use the "current" we need to change the code